lang/lang_en_fr.txt

```diff
@@ -892,7 +892,7 @@
 
 #c=20 r=4
 "Please unload the filament first, then repeat this action."
-"\x00"
+"Veuillez d'abord decharger le filament, puis repetez cette action."
```

Comment:
I believe that this translation could be shortened to `SVP, decharge le filament, puis faire cela encore`

Reply:
@awenelo Have you seen any other french translation that can be shorten/optimized?

Reply:
@awenelo Can you review other French translations as well
```
#MSG_WIZARD_REPEAT_V2_CAL c=20 r=7
"Do you want to repeat last step to readjust distance between nozzle and heatbed?"
"Voulez-vous repeter la derniere etape pour reajuster la distance entre la buse et le plateau chauffant?"
```
We will discuss these changes with the translators.

Reply:
> I believe that this translation could be shortened to SVP, dechargez le filament, puis faire cela encore
> Edit: missed a "z" in "dechargez"

SVP, dechargez le filament et recommencer

Reply:
or "SVP, dechargez le filament et reessayez

Reply:
> @awenelo Can you review other French translations as well
> #MSG_WIZARD_REPEAT_V2_CAL c=20 r=7
> "Do you want to repeat last step to readjust distance between nozzle and heatbed?"
> "Voulez-vous repeter la derniere etape pour reajuster la distance entre la buse et le plateau chauffant?"
> 
> We will discuss these changes with the translators.

"Voulez-vous recommencer l'etape pour reajuster la distance entre la buse et le plateau chauffant?"

Reply:
`renouveler` could also be translated to "renew", I think a better option/shorter would be `refaire`

Reply:
> SVP, dechargez le filament et reessayez

I take this as you both agreed on that.

Reply:
It is `"Voulez-vous refaire l'etape pour reajuster la hauteur entre la buse et le plateau?"
` for #MSG_WIZARD_REPEAT_V2_CAL c=20 r=7